### ISSUE TITLE
Right clicking causes "lockImageClickTime is not defined" errors

### ISSUE THREAD

**Issue by marpe** [extesy/hoverzoom#760]
**Describe the bug**
Right clicking on any page causes "Uncaught ReferenceError: lockImageClickTime is not defined ([hoverzoom.js:775](https://github.com/extesy/hoverzoom/blob/adff58052e2ddd35fc48dd6932304831797b626f/js/hoverzoom.js#L777))" errors to be printed to the dev console.

Stack trace
```
hoverzoom.js:775 Uncaught ReferenceError: lockImageClickTime is not defined
    at HTMLDocument.documentContextMenu (hoverzoom.js:775)
    at HTMLDocument.dispatch (jquery-3.2.1.js:5206)
    at HTMLDocument.elemData.handle (jquery-3.2.1.js:5014)
```

**To Reproduce**
1. Go to e.g google.com
2. Open chrome devtools
3. Right click
4. See error printed to console

**Desktop (please complete the following information):**
 - OS: Win 10
 - Browser: Version 91.0.4472.164 (Official Build) (64-bit)
 - Version 1.0.176

None of the functionality I use seem to be affected, so the plugin still works for me. Adding the site to the excluded list doesn't help.

**Comment by extesy** [comment]
@EhsanKia Can you take a look at this, please? Seems related to your recent PR.

**Comment by EhsanKia** [comment]
Huh, I'm seeing this too now on the live version, must've missed something on my PR. Sending fix.

**Comment by extesy** [comment]
Fixed in #766
